sharding-core/sharding-core-execute/src/main/java/org/apache/shardingsphere/core/execute/metadata/TableMetaDataLoader.java

```diff
@@ -199,7 +199,7 @@ private boolean isTableExist(final Connection connection, final String catalog,
     
     private Collection<String> getLogicIndexes(final Connection connection, final String catalog, final String actualTableName) throws SQLException {
         Collection<String> result = new HashSet<>();
-        try (ResultSet resultSet = connection.getMetaData().getIndexInfo(catalog, catalog, actualTableName, false, false)) {
+        try (ResultSet resultSet = connection.getMetaData().getIndexInfo(catalog, null, actualTableName, false, false)) {
```

Comment:
Any performance issue here if use `null` to instead of catalog name if database is not Oracle?

Reply:
I think it will make a mistake, for some database, e.g PostgreSQL, the second param is useful.

Reply:
I read mysql-connector-java-5.1.48.java and ojdbc8.jar and postresql-42.2.8.jar,the result is
1.mysql is not using the  getIndexInfo's second parameter
2.oracle is using second parameter as owner,like this "select * from all_tables where owner=?"
3.postresql is using second parameter as tablespace_name,like this
```
if (schema != null && !schema.isEmpty()) {
        sql += " AND n.nspname = " + escapeQuotes(schema);
}
```
so i think that 
1.if use mysql ,the second parameter is null
2.if use oracle that the second parameter is username
3.if use postresql that the second parameter is nspname (tablespace_name)